apphost-framework-lookup/test.json

```diff
@@ -7,6 +7,7 @@
   "type": "bash",
   "cleanup": true,
   "ignoredRIDs":[
+    "linux-s390x"
```

Comment:
This tests does a publish using `linux-s390x` on Linux. That's not supported, so disable this test.